docs/usage/actions.rst

```diff
@@ -0,0 +1,111 @@
+.. _actions:
+
+======================
+Handling action errors
+======================
```

Comment:
There is a lot of code in this section; I wonder if we should make it a part of the library itself instead. 

It seems like a general theme: if action failed, should the response be considered a success or a failure? The current default is success, but it seems that making it a failure is better (with an option to get back), as it allows to detect more issues.

Reply:
https://github.com/scrapy-plugins/scrapy-zyte-api/pull/157/commits/deedfff9eae1eeaf957bc7619f3059e6ce559370 does this for the middleware. Do we want to do something about the cache policy, or is it OK to keep that documentation-only?